### PR TITLE
[FSDE-31409] Add support for additional messaging below text input components

### PR DESCRIPTION
**Pull request by pedrotaglialatela-lumen**
https://ctl.atlassian.net/browse/FSDE-31389

### CODE REVIEW

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/custom-elements/src/components/helper-message/helper-message.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+:host(chi-label) {
```
Comment (jllr) [inline src/custom-elements/src/components/helper-message/helper-message.scss:1]: @pedrotaglialatela-lumen if the component is helper-message, for what is needed chi-label?

File: src/custom-elements/src/components/helper-message/helper-message.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Component, Element, Prop, h } from '@stencil/core';
+import { ChiStates, STATES } from "../../constants/states";
+import { FORM_CLASSES, STATUS_CLASS } from "../../constants/classes";
+
+@Component({
+  tag: 'helper-message',
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/helper-message/helper-message.tsx:6]:
```suggestion
  tag: 'chi-helper-message',
```

File: src/custom-elements/src/components/text-input/text-input.tsx [hunk 3]
```diff
@@ -194,14 +198,19 @@ export class TextInput {
...
     const spinner = this.spinner && <chi-spinner size={spinnerSizeMapping[this.size] || 'sm'} />;
-
-    const input = this.iconLeft || this.iconRight || this.spinner ?
-      <div class={`chi-input__wrapper ${iconClasses}`}>
-        {inputElement}
-        {iconLeft}
-        {iconRight}
-        {spinner}
-      </div> : inputElement;
+    const helperMessage = this.helperMessage && <helper-message state={this.state}>{this.helperMessage}</helper-message>;
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/text-input/text-input.tsx:201]:
```suggestion
    const helperMessage = this.helperMessage && <chi-helper-message state={this.state}>{this.helperMessage}</helper-message>;
```

File: src/custom-elements/src/components/helper-message/helper-message.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+:host(helper-message) {
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/helper-message/helper-message.scss:1]:
```suggestion
:host(chi-helper-message) {
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/custom-elements/src/components/helper-message/helper-message.scss (hunk 4 above)
Comment (pedrotaglialatela-lumen) [reply src/custom-elements/src/components/helper-message/helper-message.scss]: Done!!

File: src/custom-elements/src/components/text-input/text-input.tsx (hunk 3 above)
Comment (pedrotaglialatela-lumen) [reply src/custom-elements/src/components/text-input/text-input.tsx]: Done!!

File: src/custom-elements/src/components/helper-message/helper-message.tsx (hunk 2 above)
Comment (pedrotaglialatela-lumen) [reply src/custom-elements/src/components/helper-message/helper-message.tsx]: Done. Good catch. Thank you, Jose! 👍

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**